test/shared/src/main/scala/zio/test/diff/Delta.scala

```diff
@@ -0,0 +1,28 @@
+package zio.test.diff
+
+import zio.test.diff.Delta.TYPE.{ CHANGE, DELETE, INSERT }
+import zio.test.diff.Delta._
+
+private[diff] case class Delta[T](`type`: TYPE, original: Chunk[T], revised: Chunk[T])
```

Comment:
Maybe make this more Scala idiomatic. Like `diffType` instead of `type` so you don't need to use the back ticks, make `Type` becomes `DiffType` and then the case objects can go to `Change`, `Delete`, etc...